gaia-header.js

```diff
@@ -86,8 +86,7 @@ module.exports = component.register('gaia-header', {
     // Elements
     this.els = {
       actionButton: this.shadowRoot.querySelector('.action-button'),
-      buttons: this.querySelectorAll('button, a'),
-      titles: this.querySelectorAll('h1')
+      titles: this.getElementsByTagName('h1')
```

Comment:
getElementsByTagName returns a live list, so it's always uptodate. It's also really faster to call initially, but it's of course slower each time we access its elements. Because have generally only 1 element I think it's an acceptable tradeoff given the improvements.

I see that "buttons" is not used at all so I remove it altogether.